cover.tex

```diff
@@ -0,0 +1,47 @@
+\input{command.tex}
+\usepackage{parskip}
+
+\newcommand{\applyteam}{Event Storage Platform}
+\newcommand{\applycompany}{DataDog}
+\newcommand{\applyrole}{Senior Software Engineer}
+
+\begin{document}
+
+\input{header.tex}
+
+To whom it may concern,
+
+I am writing to express my interest in applying to the \applyteam\ team at \applycompany\ as a \applyrole.
+With over a decade as a full stack engineer, several years of backend focused work, and a track record for
+mentoring and growing top talent, I believe I would be a valuable asset to not only the team, but the
+organization as a whole.
+
+During my years at BlueCross BlueShield of TN and HealthcareSource, I was instrumental as a full stack
+engineer. With active involvment with the product teams, I was able to not only hone my technical skills,
+but also learn how to balance engineering and business needs.
+
+As my career progressed, I began to focus on the pure engineering challenges afforded
+to backend development. I've excelled at leading and maintaining 3rd party integrations, full redesigns of
+asynchronous job systems, and various other backend projects. The majority of my tenure at ThirtyMadison
+involved working on foundation and core services teams where my work was entirely backend focused. In the
+latter year, I became a leader of our Architecture Review Group.
+
+With a specific note on DataDog, ThirtyMadison uses it extensively. I was not only a consumer of the data
+we collected to establish dashboard, monitors, and metrics, but also enhanced our collection for analysis.
```

Comment:
"but was also" - makes the distinction between your list in the first half of this sentence more pronounced from the second half

Reply:
not sure what you're going for here.

Reply:
"**was** not only [...] but **was** also [...]"

It lets the reader know the sentence has transitioned, like either/or, neither/nor, etc.

Since the firs half of the sentence contains a list, it lets the reader know you're no longer enumerating and continuing on with something started at the beginning of the sentence.

I don't think this is a grammatical issue, in fact I think grammar checks might underline it, but it makes it easier to read imo. Up to you!